prac_07/project.py

```diff
@@ -0,0 +1,30 @@
+"""Project class"""
+
+
+class Project:
+    """"""
+
+    def __init__(self, name="", start_date="", priority=0, cost_estimate=0.0, completion=0):
+        """Initialise a Project."""
+        self.name = name
+        self.start_date = start_date
+        self.priority = priority
+        self.cost_estimate = cost_estimate
+        self.completion = completion
+
+    def __repr__(self):  # check repr vs str
+        """Represent Project as string."""  # TODO: fix description!
+        return f"{self.name}\t{self.start_date}\t{self.priority}\t{self.cost_estimate:.1f}\t{self.completion}"
+
+    def __str__(self):
+        """Represent Project as string."""  # TODO: fix description!
+        return f"{self.name}, start: {self.start_date}, priority {self.priority}, estimate: ${self.cost_estimate:,.2f}, " \
+               f"completion: {self.completion}%"
+
+    def __lt__(self, other):
```

Comment:
Could this be used to filter by date as well?

Reply:
Yes, good point! That would make things so much clearer!